core/src/main/java/com/graphhopper/util/Parameters.java

```diff
@@ -191,9 +191,9 @@
      */
     public static final class DETAILS {
 
-        private static final String DETAILS_PREFIX = "details.";
```

Comment:
Mhm, I think we have to differentiate between two different concepts here. Either we use the old way of requesting the PathDetails or the new array style that was proposed by you.

> Yes, the variable should be named DETAILS_PREFIX
> and the dot is missing, so yes, a revert is best: DETAILS_PREFIX="details." or are you using this elsewhere where the dot is bad?

The usage of the variable just completely changed, it's not used as a prefix anymore. Right now this is used for the request in the form `detail=average_speed`. If we go back to the old request style we would write `details.average_speed=true`. Then we would add the dot and rename it again to DETAILS_PREFIX. Let me know which style you prefer.